docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -442,6 +446,10 @@ ref-spec is:
 
 column-list is:
    column-name[, column-name]...
+
+index-name is:

```

Comment:
I don't think I would make this distinction in the syntax. There is no syntactic difference between what you're calling a "normal index" and a unique index. And all index names are in the same name space, whether or not they are unique. Consider deleting these lines.

Reply:
Thanks @DaveBirdsall, I've deleted. :)